src/AuthApi/AuthTokenProvider.php

```diff
@@ -105,6 +105,21 @@ public static function create($userIdentifier, $secretKey, $logger = null)
         return new self($userIdentifier, $secretKey, $client, $logger);
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getAccessToken()
+    {
+        if (!$this->tokenExists()) {
+            $this->requestTimestamp = time();
+            $this->data = $this->authenticate();
+            $this->getLogger()->debug(var_export($this->data, true));
```

Comment:
Pls remove such loggging before merge to master. Method `sendRequest` already has logging